root/static/scripts/release-editor/fields.js

```diff
@@ -358,6 +358,7 @@ class Medium {
     var tracks = data.tracks;
     this.tracks = ko.observableArray(utils.mapChild(this, tracks, Track));
     this.tracksUnknownToUser = ko.observable(false);
+    this.tracksWereUnknownToUser = false;
```

Comment:
How come this adds a new variable instead of just using `tracksUnknownToUser`?

Reply:
Because we care about whether the tracks were unknown *at first*, but tracksUnknownToUser is updated when the user unchecks the checkbox.